### PR TITLE
AMB-188 implement proxy

### PR DESCRIPTION
**Pull request by RyanThomas1214**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: docker/reasonable-adjustments/src/config.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        }
+  },
+  "list/:listID": {
+    "latency": 20,
+    "verbs": ["put"],
+    "responses": {
+          "put": {"httpStatus": 200, "mockFile": "updateList.json"}
+        }
+    },
+   "removerarecord": {
```
Comment (artronics) [inline docker/reasonable-adjustments/src/config.json:64]: Can we rename this to cameCase?

File: proxies/live/apiproxy/proxies/default.xml [hunk 2]
```diff
@@ -19,6 +18,100 @@
...
+    <Flow name="list-PUT">
+      <Request>
+        <Step>
+          <Name>RaiseFault.400BadRequest</Name>
+          <Condition>((request.content = "") and (request.header.If-Match = null))</Condition>
+        </Step>
+      </Request>
+      <Condition>((proxy.pathsuffix MatchesPath "/list/*") and (request.verb = "PUT"))</Condition>
+    </Flow>
+    <Flow name="removerarecord-POST">
```
Comment (artronics) [inline proxies/live/apiproxy/proxies/default.xml:102]: Also this policy naming doesn't follow our naming convention.

File: docker/reasonable-adjustments/src/config.json (hunk 1 above)
Comment (adrianclare) [reply docker/reasonable-adjustments/src/config.json:64]: This file isn't used; also the endpoint isn't came case

File: docker/reasonable-adjustments/src/mocks/consentPUT.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "Date": "Tue, 23 Jul 2018 11:00:00 GMT",
+    "Last-Modified": {},
+    "ETag": "resourceVID",
```
[the remark this answers is not in the document]
Comment (adrianclare) [reply docker/reasonable-adjustments/src/mocks/consentPUT.json:4]: Is it worth making the change? This is going to be redundant soon once we have a real back-end configured; the change is worth making for the actual sandbox though

[the remark this answers is not in the document]
Comment (artronics) [reply docker/reasonable-adjustments/src/mocks/consentPUT.json:4]: Yeah it's not the sandbox. My bad

[the remark this answers is not in the document]
Comment (adrianclare) [reply docker/reasonable-adjustments/src/mocks/consentPUT.json:4]: Might be best creating a new ticket for improvements for the sandbox as this is just focusing on the real proxy.

**Review by artronics — APPROVED** [review]

**Pull request merged**